packages/arb-token-bridge-ui/src/components/common/Transition.tsx

```diff
@@ -1,53 +1,50 @@
-import React, { Fragment } from 'react'
-import { Transition as HeadlessUITransition } from '@headlessui/react'
+import { Transition as HeadlessUiTransition } from '@headlessui/react'
+import { PropsWithChildren } from 'react'
+import { twMerge } from 'tailwind-merge'
 
-type TransitionDuration = 'fast' | 'normal'
+type TransitionSpeed = 'slow' | 'normal' | 'fast'
 
-function getDurationNumbers(
-  duration: TransitionDuration
-): [enterDuration: number, leaveDuration: number] {
-  switch (duration) {
-    case 'fast':
-      return [150, 100]
-
-    case 'normal':
-      return [300, 200]
-  }
+type TransitionOptions = {
+  enterSpeed?: TransitionSpeed
+  leaveSpeed?: TransitionSpeed
+  unmountOnLeave?: boolean
 }
 
-export function getTransitionProps(duration: TransitionDuration = 'fast') {
-  const [enterDuration, leaveDuration] = getDurationNumbers(duration)
+type TransitionProps = PropsWithChildren<{
+  isOpen?: boolean
+  options?: TransitionOptions
+  className?: string
+}>
 
-  return {
-    enter: `transition ease-out duration-${enterDuration}`,
-    enterFrom: 'opacity-0',
-    enterTo: 'opacity-100',
-    leave: `transition ease-out duration-${leaveDuration}`,
-    leaveFrom: 'opacity-100',
-    leaveTo: 'opacity-0'
+function getDurationClassName(speed: TransitionSpeed) {
+  switch (speed) {
+    case 'slow':
+      return 'duration-[1000ms]'
```

Comment:
please add it to tailwind config otherwise it doesn't get compiled by tailwind because it isn't used inline in the DOM

Reply:
yea I was meant to go back to it, thanks for reminding